docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+version: '3'
+
+services:
+
+  elasticsearch1:
+    image: docker.elastic.co/elasticsearch/elasticsearch:5.3.1
+    container_name: elasticsearch1
```

Comment:
Why are we specifying the container name?  Doesn't this break running multiple stacks simultaneously?